### ISSUE TITLE
Captive portal not popping up

### ISSUE THREAD

**Issue by anoo2niem** [wifiphisher/wifiphisher#668]
Hello, 

I'm trying to setup a fake hotspot which pops up captive portal and then passes me the user credentials.
I've tried wifiphisher on my current laptop, and it works flawless.

I recently bought a raspberry pi 3 model b. I'm running kali on it and tried to install wifiphisher on it by apt-get install wifiphisher. It is the 1.1 version which hadn't Oauth login option. so i removed it again an did a git clone from the current version. When installing i ran into some errors. So i deleted it again.
I downloaded the 1.2 version, unzipped it and then ran installer.sh. This worked and had a Oauth option.

I have setup an ad-hoc network on the internal wifi so i can connect to it with ssh from my iphone. i ran wifiphisher with oauth option. It worked and passed me the credentials. Thats the last time it worked. it worked only once. After that it never showed up a captive portal again on different ios devices. So i tried to install v1.3 again. I managed to install it without errors, but still the problem remains. It sets up the hotspot. Devices can connect to it, and signs them an ip address, but the captive portal never shows up again. 

I've tried to setup my wlan interface manually in monitor mode before running by airmon-ng start wlan1, and restarting hostapd manually by service hostapd restart. Don't know if this is relevant but thats as far my knowledge goes. 

Do u have any suggestions? Can i provide u some more relevant data? If yes which data and with what commands?

Thanks in advance!

**Comment by anakin1028** [comment]
Can you post the capture from the fakeap interface and we can see why `wifiphisher` doesn't redirect to CaptivePortal.
Also can you try manaully to enter a web page and see if the device redirect to the phishing page?

**Comment by anoo2niem** [comment]
Jamming devices: 

DHCP Leases: 
1499570911 bc:54:36:3t:c9:70 10.0.0.39 Anoo 01:bc:54:36:3t:c9:70

HTTP requests: 

When i enter a web page manually nothing happens, even if i do http://10.0.0.1 the page keeps loading.

Thanks for your response

**Comment by anoo2niem** [comment]
i've tried to look at the status of dnsmasq with `service dnsmasq status`. 
And it gives me the following: `Unit dnsmasq.service could not be found.`

I did `whereis dnsmasq` and it gave me `dnsmasq: /usr/sbin/dnsmasq /usr/share/man/man8/dnsmasq.8.gz`

Maybe this could have something to do with it? I think dnsmasq handles the captive portal, i could be wrong though.

**Comment by anoo2niem** [comment]
I've got this recently:
'File "bin/wifiphisher", line 12, in <module>
    run()
  File "/root/wifiphisher-1.3/wifiphisher/pywifiphisher.py", line 776, in run
    engine.start()
  File "/root/wifiphisher-1.3/wifiphisher/pywifiphisher.py", line 759, in start
    proc = check_output(['tail', '-5', '/var/lib/misc/dnsmasq.leases'])
  File "/usr/lib/python2.7/subprocess.py", line 212, in check_output
    process = Popen(stdout=PIPE, *popenargs, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1010, in _execute_child
    _close_in_parent(c2pwrite)
  File "/usr/lib/python2.7/subprocess.py", line 902, in _close_in_parent
    os.close(fd)
OSError: [Errno 9] Bad file descriptor'
`

**Comment by anakin1028** [comment]
Can you use wireshark or tcpdump capture the frames on the fakeAP interface? You can post it on this thread and we can see if wifiphisher redirect the http request to CaptivePortal.

**Comment by anoo2niem** [comment]
`listening on wlan1, link-type EN10MB (Ethernet), capture size 262144 bytes
16:38:56.864036 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:39:11.381705 bc:54:36:3b:c7:77 (oui Unknown) > Broadcast Null Unnumbered, xid, Flags [Response], length 6: 01 00
16:39:11.470035 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 8
16:39:11.470381 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 8
16:39:11.628029 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:11.628448 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:11.947130 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:39:11.947441 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:39:12.633371 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:12.633752 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:12.948272 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 2 group record(s), length 48
16:39:12.948608 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 2 group record(s), length 48
16:39:13.949413 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:39:13.949743 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:39:14.636470 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:14.636806 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:15.003050 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [2q] [2n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. (147)
16:39:15.003360 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [2q] [2n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. (147)
16:39:15.258682 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [2q] [2n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. (147)
16:39:15.259007 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [2q] [2n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. (147)
16:39:15.513395 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [2q] [2n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. (147)
16:39:15.513771 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [2q] [2n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. (147)
16:39:15.747303 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:39:15.747642 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:39:15.768445 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:15.768761 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:16.772308 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:16.772687 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:18.261987 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:18.262294 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:18.642263 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:18.642633 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:20.341125 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:39:20.341426 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:39:22.184524 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:22.184873 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 6/0/4 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe (421)
16:39:26.654086 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:26.654519 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:28.203648 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:28.204053 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:29.908742 IP 10.0.0.1.bootps > 10.0.0.39.bootpc: BOOTP/DHCP, Reply, length 300
16:39:29.913242 IP 10.0.0.1.bootps > 10.0.0.39.bootpc: BOOTP/DHCP, Reply, length 300
16:39:30.979179 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:30.979532 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:39:30.994140 IP 10.0.0.1.bootps > 10.0.0.39.bootpc: BOOTP/DHCP, Reply, length 300
16:39:31.115411 ARP, Request who-has 10.0.0.39 tell 0.0.0.0, length 28
16:39:31.115689 ARP, Request who-has 10.0.0.39 tell 0.0.0.0, length 28
16:39:31.461338 ARP, Request who-has 10.0.0.39 tell 0.0.0.0, length 28
16:39:31.461657 ARP, Request who-has 10.0.0.39 tell 0.0.0.0, length 28
16:39:31.810773 ARP, Request who-has 10.0.0.39 tell 0.0.0.0, length 28
16:39:31.811159 ARP, Request who-has 10.0.0.39 tell 0.0.0.0, length 28
16:39:32.158496 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:39:32.158851 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:39:32.504598 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:39:32.504887 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:39:32.852323 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:39:32.852686 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:39:32.853074 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:39:32.853356 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:39:32.853776 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:39:32.854190 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:32.854467 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.068382 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.068701 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.372257 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:39:33.372613 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:39:33.373091 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:39:33.373380 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:39:33.373671 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:39:33.374108 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:39:33.374423 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:39:33.382754 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:39:33.394007 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.394296 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.557303 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.557668 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.558117 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.558433 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.718157 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.718517 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:33.890319 IP 10.0.0.39 > 224.0.0.2: igmp leave 224.0.0.251
16:39:33.890718 IP 10.0.0.39 > 224.0.0.2: igmp leave 224.0.0.251
16:39:33.891187 IP 10.0.0.39 > 224.0.0.251: igmp v2 report 224.0.0.251
16:39:33.891497 IP 10.0.0.39 > 224.0.0.251: igmp v2 report 224.0.0.251
16:39:33.891918 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.892241 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.892670 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:33.892952 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:39:34.043329 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:34.043686 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:39:34.067836 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:34.068271 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:34.068730 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:34.069056 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:34.169954 IP 10.0.0.39.57683 > 10.0.0.1.domain: 38346+ A? i.w.inmobi.com. (32)
16:39:34.312033 IP 10.0.0.39.61015 > 10.0.0.1.domain: 65264+ A? apple.com. (27)
16:39:34.323364 IP 10.0.0.39.65515 > 10.0.0.1.domain: 63721+ A? p22-keyvalueservice.icloud.com. (48)
16:39:34.340021 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:39:35.074940 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:35.075285 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:35.075736 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:35.076029 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:35.263351 IP 10.0.0.39.57683 > 10.0.0.1.domain: 38346+ A? i.w.inmobi.com. (32)
16:39:37.083372 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:37.083759 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:37.084278 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:37.084592 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:37.430263 IP 10.0.0.39.61015 > 10.0.0.1.domain: 65264+ A? apple.com. (27)
16:39:37.430774 IP 10.0.0.39.65515 > 10.0.0.1.domain: 63721+ A? p22-keyvalueservice.icloud.com. (48)
16:39:37.431074 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:39:38.299650 IP 10.0.0.39.57683 > 10.0.0.1.domain: 38346+ A? i.w.inmobi.com. (32)
16:39:41.080437 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:41.080882 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:41.081396 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:41.081684 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) PTR Anoo.local., (Cache flush) A 10.0.0.39 (487)
16:39:46.341574 IP 10.0.0.39.61015 > 10.0.0.1.domain: 65264+ A? apple.com. (27)
16:39:46.351243 IP 10.0.0.39.65515 > 10.0.0.1.domain: 63721+ A? p22-keyvalueservice.icloud.com. (48)
16:39:46.369859 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:39:47.311334 IP 10.0.0.39.57683 > 10.0.0.1.domain: 38346+ A? i.w.inmobi.com. (32)
16:40:11.220163 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:11.220510 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:40:11.220824 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:11.222791 IP 10.0.0.39.62853 > 10.0.0.1.domain: 58504+ A? i.w.inmobi.com. (32)
16:40:12.219942 IP 10.0.0.39.62853 > 10.0.0.1.domain: 58504+ A? i.w.inmobi.com. (32)
16:40:13.495222 IP 10.0.0.39.61015 > 10.0.0.1.domain: 65264+ A? apple.com. (27)
16:40:13.495696 IP 10.0.0.39.65515 > 10.0.0.1.domain: 63721+ A? p22-keyvalueservice.icloud.com. (48)
16:40:13.495992 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:40:15.241578 IP 10.0.0.39.62853 > 10.0.0.1.domain: 58504+ A? i.w.inmobi.com. (32)
16:40:24.259500 IP 10.0.0.39.62853 > 10.0.0.1.domain: 58504+ A? i.w.inmobi.com. (32)
16:40:24.601265 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 1/0/1 PTR DisplayRecorder op  Anoo._http._tcp.local. (96)
16:40:24.601680 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 1/0/1 PTR DisplayRecorder op  Anoo._http._tcp.local. (96)
16:40:24.602108 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 1/0/1 PTR DisplayRecorder op  Anoo._http._tcp.local. (96)
16:40:24.602473 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 1/0/1 PTR DisplayRecorder op  Anoo._http._tcp.local. (96)
16:40:30.088122 bc:54:36:3b:c7:77 (oui Unknown) > Broadcast Null Unnumbered, xid, Flags [Response], length 6: 01 00
16:40:30.206886 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:30.207207 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:40:30.208232 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:40:30.208575 IP 0.0.0.0.bootpc > 255.255.255.255.bootps: BOOTP/DHCP, Request from bc:54:36:3b:c7:77 (oui Unknown), length 300
16:40:30.212910 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 8
16:40:30.213225 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 8
16:40:30.217256 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:40:30.217575 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:40:30.218735 IP 10.0.0.1.bootps > 10.0.0.39.bootpc: BOOTP/DHCP, Reply, length 300
16:40:30.225013 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:30.225349 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:30.295872 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:30.296322 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:40:30.296879 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:30.298896 IP 10.0.0.39.49464 > 10.0.0.1.domain: 63214+ A? 9-courier.push.apple.com. (42)
16:40:30.442895 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:40:30.443249 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:40:30.443727 IP 10.0.0.39.61015 > 10.0.0.1.domain: 65264+ A? apple.com. (27)
16:40:30.444142 IP 10.0.0.39.65515 > 10.0.0.1.domain: 63721+ A? p22-keyvalueservice.icloud.com. (48)
16:40:30.444490 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:40:30.444832 IP 10.0.0.39.62853 > 10.0.0.1.domain: 58504+ A? i.w.inmobi.com. (32)
16:40:30.619151 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:30.619486 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:30.619936 IP6 :: > ff02::1:ff3b:7abe: ICMP6, neighbor solicitation, who has fe80::8b6:85e2:943a:7abe, length 24
16:40:30.620272 IP6 :: > ff02::1:ff3b:7abe: ICMP6, neighbor solicitation, who has fe80::8b6:85e2:943a:7abe, length 24
16:40:30.966140 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:30.966429 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:31.140969 IP 10.0.0.39 > 224.0.0.2: igmp leave 224.0.0.251
16:40:31.141334 IP 10.0.0.39 > 224.0.0.2: igmp leave 224.0.0.251
16:40:31.141838 IP 10.0.0.39 > 224.0.0.251: igmp v2 report 224.0.0.251
16:40:31.142199 IP 10.0.0.39 > 224.0.0.251: igmp v2 report 224.0.0.251
16:40:31.142561 IP 10.0.0.39 > 224.0.0.2: igmp leave 224.0.0.251
16:40:31.142844 IP 10.0.0.39 > 224.0.0.2: igmp leave 224.0.0.251
16:40:31.143253 IP 10.0.0.39 > 224.0.0.251: igmp v2 report 224.0.0.251
16:40:31.143532 IP 10.0.0.39 > 224.0.0.251: igmp v2 report 224.0.0.251
16:40:31.314185 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:31.314602 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:31.315059 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:31.315338 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:40:31.315709 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:31.321602 IP 10.0.0.39.49464 > 10.0.0.1.domain: 63214+ A? 9-courier.push.apple.com. (42)
16:40:31.443541 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:40:31.443916 IP6 fe80::8b6:85e2:943a:7abe > ff02::16: HBH ICMP6, multicast listener report v2, 1 group record(s), length 28
16:40:31.595182 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:31.595619 ARP, Request who-has 169.254.255.255 tell 10.0.0.39, length 28
16:40:31.596113 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:40:31.596401 ARP, Request who-has 10.0.0.39 tell 10.0.0.39, length 28
16:40:31.974223 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:31.974547 ARP, Reply 10.0.0.1 is-at 00:c0:ca:92:33:2f (oui Unknown), length 28
16:40:31.974912 ARP, Request who-has 10.0.0.1 tell 10.0.0.39, length 28
16:40:33.531192 IP 10.0.0.39.61015 > 10.0.0.1.domain: 65264+ A? apple.com. (27)
16:40:33.531742 IP 10.0.0.39.65515 > 10.0.0.1.domain: 63721+ A? p22-keyvalueservice.icloud.com. (48)
16:40:33.532168 IP 10.0.0.39.54024 > 10.0.0.1.domain: 58687+ A? www.apple.com. (31)
16:40:33.532524 IP 10.0.0.39.62853 > 10.0.0.1.domain: 58504+ A? i.w.inmobi.com. (32)
16:40:34.405846 IP 10.0.0.39.49464 > 10.0.0.1.domain: 63214+ A? 9-courier.push.apple.com. (42)
16:40:35.105326 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:40:35.105624 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:40:35.284353 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:40:35.284677 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:40:35.285125 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:40:35.285462 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QU)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QU)? Anoo.local. ANY (QU)? Anoo.local. (169)
16:40:35.627954 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.628409 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.628937 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.629294 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.806852 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.807234 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.807690 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:35.807966 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0 [3q] [3n] [1au] ANY (QM)? DisplayRecorder op  Anoo._http._tcp.local. ANY (QM)? Anoo.local. ANY (QM)? Anoo.local. (169)
16:40:36.157271 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:36.157651 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:36.158103 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:36.158411 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:37.056214 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:37.056642 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:37.057147 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:37.057488 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:38.156997 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:38.157358 IP 10.0.0.39.mdns > 224.0.0.251.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:38.157814 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:38.158117 IP6 fe80::8b6:85e2:943a:7abe.mdns > ff02::fb.mdns: 0*- [0q] 8/0/5 (Cache flush) TXT "path=/" "liveVideoPath=/video" "server=DisplayRecorder" "screenshotPath=/screenshot", PTR _http._tcp.local., PTR DisplayRecorder op  Anoo._http._tcp.local., (Cache flush) SRV Anoo.local.:4452 0 0, (Cache flush) PTR Anoo.local., (Cache flush) PTR Anoo.local., (Cache flush) AAAA fe80::8b6:85e2:943a:7abe, (Cache flush) A 10.0.0.39 (487)
16:40:39.275404 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
16:40:39.275755 IP6 fe80::8b6:85e2:943a:7abe > ip6-allrouters: ICMP6, router solicitation, length 16
`

**Comment by anoo2niem** [comment]
I've managed to figure out the cause of this issue.
When the configuration in /etc/network/interfaces is like this
```
auto lo
iface lo inet loopback

auto eth0
iface eth0 inet dhcp`
```

In this case i can enable wifiphisher with keyboard attached to the respberry pi, and connected to a screen. Then the captive portal pops up on my iphone.

When i use this configuration for /etc/network/interfaces 
```
auto lo
iface lo inet loopback

auto eth0
iface eth0 inet dhcp

auto wlan0
iface wlan0 inet static
        address 10.0.0.1
        netmask 255.255.255.0
        wireless-channel 1
        wireless-essid PiFi
        wireless-mode ad-hoc
        wireless-key ***
```
I use this config to set up an ad-hoc network on the wlan0 interface from the raspberry pi, so i can ssh to it whenever i want, even without an actual internet connection. When this configuration is used i can start wifiphisher, connect to the fakeAP, it gives me an ip but there are no HTTP request. So no captive portal.
`
I've tested it several times so im pretty sure thats the issue. 

Is there anyway i can "fix" this?

**Comment by anoo2niem** [comment]
Problem is solved by changing the ip from my ad-hoc config to 192.168.0.1

I had my ad-hoc network setup in same ip range as wifiphisher, guess they were interfering.

**Comment by anakin1028** [comment]
Cool !

**Comment by anakin1028** [comment]
About another issue, does that happen when you exit wifiphisher?

**Comment by anoo2niem** [comment]
I'm not sure if i interpret your question correctly, but if u meant the error output mentioned above, then yes it happened after exiting wifiphisher.

**Comment by Lucifer20001** [comment]
> Problem is solved by changing the ip from my ad-hoc config to 192.168.0.1
> 
> I had my ad-hoc network setup in same ip range as wifiphisher, guess they were interfering.

hey , i am little new here i ran into some issues while installing wifiphisher and the roguehostapd with it, and i am facing the same issue you had can you guide me a little on how to troubleshoot it ?
